### DISCUSSION TITLE
Is Aria DnD for Grids?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [adobe/react-spectrum#3773] [General]
I see many drag & drop Examples for Lists but how about Grids is Aria DnD usable for grids?

**Comment by reidbarber** [comment]
Yes, the DnD hooks work with grids. You can take the example in the [useDraggableCollection docs](https://react-spectrum.adobe.com/react-aria/useDraggableCollection.html#example), and use `useGrid` and `useGridState` in place of `useListbox` and `useListState`. There aren't any docs for those hooks just yet, so you may need to look through the repo for some examples.

**Reply by ivanjeremic** [reply]
Thanks, hope there will be documentation on it soon :)
